collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py

```diff
@@ -95,6 +97,13 @@ def gpu_charts(gpu):
                 ['fb_memory_used', 'used'],
             ]
         },
+        BAR_USAGE: {
+            'options': [None, 'Bar1 Memory Usage', 'MiB', fam, 'nvidia_smi.bar1_allocated', 'stacked'],
```

Comment:
```suggestion
            'options': [None, 'Bar1 Memory Usage', 'MiB', fam, 'nvidia_smi.bar1_memory_usage', 'stacked'],
```

Let's make the context more clear

Reply:
Or `bar1_memory_allocated` ? The above **MEM_USAGE** makes use of `nvidia_smi.memory_allocated`.

Reply:
I just commited with `bar1_memory_usage`, but as you want!

Reply:
> The above MEM_USAGE makes use of nvidia_smi.memory_allocated

Oh, you are right. I think that "mem_usage" is a better name, but we could make it "bar1_memory_allocated" to keep consistent with MEM_USAGE. I guess I'm ok with both variants.